src/client/app/components/unit/EditUnitModalComponent.tsx

```diff
@@ -450,6 +480,21 @@ export default function EditUnitModalComponent(props: EditUnitModalComponentProp
 							</FormFeedback>
 						</FormGroup></Col>
 					</Row>
+					{/*Disable checks box */}
+					<FormGroup>
+						<Label for='disableChecks'>{translate('disable.checks')}</Label>
+						<Input
+							id='disableChecks'
+							name='disableChecks'
+							type='select'
+							value={state.note}
+							onChange={e => handleStringChange(e)}>
+							<option value=''> Select an option </option>
+							<option value='No, only reject the bad reading(s)'> No, only reject the bad reading(s) </option>
+							<option value='No, reject all readings in batch'> No, reject all readings in batch </option>
+							<option value='Yes, do not check values'> Yes, do not check value </option>
+						</Input>
+					</FormGroup>
```

Comment:
For Disable checks, the expectation is to create an enum and options should loop through the enum and translate the enum keys to display values of the options